packages/xdl/src/project/ManifestHandler.ts

```diff
@@ -231,7 +233,31 @@ export async function getManifestResponseAsync({
   await resolveGoogleServicesFile(projectRoot, manifest);
 
   // Create the final string
-  const manifestString = await getManifestStringAsync(manifest, hostInfo.host, acceptSignature);
+  let manifestString;
+  try {
+    manifestString = await getManifestStringAsync(manifest, hostInfo.host, acceptSignature);
+  } catch (error) {
+    if (error.code === 'UNAUTHORIZED_ERROR' && manifest.owner) {
+      // Don't have permissions for siging, warn and enable offline mode.
+      addSigningDisabledWarning(
+        projectRoot,
+        `This project belongs to ${chalk.bold(
+          `@${manifest.owner}`
+        )} and you have not been granted the appropriate permissions.\n` +
+          `Please request access from an admin of @${manifest.owner} or change the \"owner\" field to an account you belong to.\n` +
+          learnMore('https://docs.expo.io/versions/latest/config/app/#owner')
+      );
+      Config.offline = true;
+      manifestString = await getManifestStringAsync(manifest, hostInfo.host, acceptSignature);
+    } else if (error.code === 'ENOTFOUND') {
+      // Got a DNS error, i.e. can't access exp.host, warn and enable offline mode.
+      addSigningDisabledWarning(projectRoot, 'Not connected to internet.');
```

Comment:
For users in countries where you might need a VPN, this could be confusing. They could have access to "the internet" but not `exp.host`. 

- Maybe rewording this to "Could not reach the Expo servers" might be better?
- Or even better, add tips for the user to manually test/fix connection (e.g. "Could not reach Expo servers, please check if you can access exp.host")

Reply:
Great point, I'll update the wording here.